helm-chart/kuberay-apiserver/templates/ingress.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.ingress.create -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: kuberay-apiserver-ingress
+spec:
+  rules:
+    - host: api.kuberay.{{ .Values.ingress.host }}
```

Comment:
is this by default included in the helm?

Reply:
If that's just for downstream. Let's remove it here? we can support it later in downstream repo